### PR TITLE
S2359

### PR DESCRIPTION
**Pull request by tomsfeldmanis**
Please also review https://github.com/finansportalen/finansportalen2/commit/407f7749fe0658958c4919d6faf39536da7008eb (initial commit for the task, messed up and didn't include it into this branch)

### CODE REVIEW

**Comment by tomsfeldmanis** [comment]
Assigning to @ssinica

File: site-cms/wordpress/.htaccess [hunk 1]
```diff
@@ -8,6 +8,8 @@ RewriteEngine On
 RewriteBase /
 RewriteRule ^index\.php$ - [L]
 
+RewriteRule ^wp-rest/([^/]+)/?([^/]*)/?$ index.php?rest-method=$1&rest-param1=$2 [L]
```
Comment (ssinica) [inline site-cms/wordpress/.htaccess]: This rule should be, probably, added to Apache configuration? Right?

File: site-cms/wordpress/wp-content/themes/finansportalen/external/js/config.prod.js [hunk 2]
```diff
@@ -29,7 +29,10 @@ requirejs.config({
         insuranceCommonApp: 'insurance/common/insurance-common-app',
 
         // travel insurance
-        travelInsuranceApp: 'insurance/travel-insurance'
+        travelInsuranceApp: 'insurance/travel-insurance',
+        
+    	// words and expressions
+        wordsAndExpressionsApp: 'words-and-expressions/words-and-expressions-app'
```
Comment (ssinica) [inline site-cms/wordpress/wp-content/themes/finansportalen/external/js/config.prod.js]: Define a module for `wordsAndExpressionsApp` in `site-cms/tools/build.js` and make sure build is working (see `example-prod-build.cmd` for details how to run build with minification enabled).

**Comment by tomsfeldmanis** [comment]
@ssinica Thanks! Prod build you mentioned was successful. About apache configuration - yes, the .htaccess file should be updated on both WordPress instances

**Comment by ssinica** [comment]
@tomsfeldmanis No, I was more thinking that we should't use `.htaccess`. From Apache manual:

> You should avoid using .htaccess files completely if you have access to httpd main server config file. Using .htaccess files slows down your Apache http server. Any directive that you can include in a .htaccess file is better set in a Directory block, as it will have the same effect with better performance.

I will remove `.htaccess` file completely and copy configuration to main Apache configuration.

**Comment by tomsfeldmanis** [comment]
@ssinica ok, makes sense

**Comment by ssinica** [comment]
@tomsfeldmanis I creates a separate task for this https://finansportalen.myjetbrains.com/youtrack/issue/S-2523.

**Pull request merged**